docs/requirements.txt

```diff
@@ -21,9 +21,7 @@ speechbrain==0.5.13
 huggingface_hub==0.11.1
 fasttext-wheel==0.9.2
 torch==1.13.1
-skorch==0.12.1
```

Comment:
Nice catch! Can confirm that `skorch` is [only imported in `tests/test_frameworks.py`](https://github.com/cleanlab/cleanlab/blob/14d1a86d4375beb17168eafdea88c84fd73c9a65/tests/test_frameworks.py#L41).